src/genie/libs/parser/nxos/show_hsrp_event_history_msgs.py

```diff
@@ -0,0 +1,133 @@
+from genie.metaparser import MetaParser
```

Comment:
Please incorporate the code from this file into `genieparser/src/genie/libs/parser/nxos/show_hsrp.py` to keep things organized. Once this code code has been added to `show_hsrp.py`, please delete this file.